src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
> Component children should use it as parent reference. Do you agree?
Obviously, for this particular example and for Este in general, it makes 100% sense to do it in the way you said. I've just overcomplicated this `refactoring` by putting my everyday `style` into it. Which for React does not make sense as the stylesheets are going to be separated anyway :)